addons/common/fnc_waitUntilAndExecute.sqf

```diff
@@ -25,8 +28,12 @@ Author:
 ---------------------------------------------------------------------------- */
 #include "script_component.hpp"
 
-params [["_condition", {}, [{}]], ["_statement", {}, [{}]], ["_args", []]];
+params [["_condition", {}, [{}]], ["_statement", {}, [{}]], ["_args", []], ["_timeout", 1E9, [0]], ["_timeoutCode", {}, [{}]]];
 
-GVAR(waitUntilAndExecArray) pushBack [_condition, _statement, _args];
+if (_timeout == 1E9) then {
```

Comment:
maybe use `isNil "_timeout"` and have no default value / default value `nil` in `params`?
No one will ever use `1E9`, but having a random high number seems weird. (Especially an integer above single point floating precision.)